Inseok_45/routes/postRouter.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const postController = require('../controllers/postController');
+
+const router = express.Router();
+
+router.post('/', postController.createPosts);
+router.get('/', postController.getAllPosts);
```

Comment:
```suggestion
router.post('', postController.createPosts);
router.get('', postController.getAllPosts);
```
마찬가지로 이외의 엔드포인트들도 모두 반영해주세요~